hasher-matcher-actioner/webapp/src/components/settings/ActionRuleFormColumns.jsx

```diff
@@ -33,7 +42,12 @@ export default function ActionRuleFormColumns({
             const newClassification = classification;
             newClassification.classification_type = e.target.value;
             onClassificationChange(newClassification);
-          }}>
+          }}
+          isInvalid={
+            showErrors &&
+            classification.classification_type === classificationTypeTBD
+          }>
+          <option value={classificationTypeTBD}> - </option>
```

Comment:
Let's change `-` to `Select...`

If the pick list was wider I'd say it should be `Select Classification Type...` following the pattern used for selecting action. But since the width is limited, `Select...` will do.

Reply:
Maybe change the default option for the action select (line 177) to just be `Select...`, too, for consistency (what you're selecting is implied, at least in the action column).